samples/highcharts/demo/combo-data-resolution/demo.details

```diff
@@ -0,0 +1,10 @@
+---
+name: 'Same data, different intervals'
+authors:
+  - Markus Barstad
+js_wrap: b
+tags:
```

Comment:
I think, for Demo pages demos we need also `alt_text` option. I know, that most demos don't have it, but we should add it to new demos.